django/test/utils.py

```diff
@@ -170,6 +171,136 @@ def teardown_test_environment():
     del mail.outbox
 
 
+class TestDatabaseSetup:
```

Comment:
Could we move that back to the function, what benefits does switching to a class where `setup` is call on provides?

Reply:
BTW @charettes , is it ok if I keep re-requesting reviews when you review something? I'm hesitant about it